src/renderer/tvgRender.h

```diff
@@ -153,6 +154,36 @@ struct RenderStroke
         trim = rhs.trim;
     }
 
+    bool adjustTrim(float& begin, float& end) const
```

Comment:
Please use this name:
```
bool strokeTrim(float& begin, float&end) const
```

- I hope you know that ThorVG doesn't stick to using the verb-objective style. 
- Regardless of any internal adjustments. just knowing the valid return is enough.